packages/tools/lib/create-new-component/index.js

```diff
@@ -1,8 +1,32 @@
 const fs = require("fs");
+const path = require("path");
 const prompts = require("prompts");
 const jsFileContentTemplate = require("./jsFileContentTemplate.js");
 const tsFileContentTemplate = require("./tsFileContentTemplate.js");
 
+/**
+ * Hyphanates the given PascalCase string.
+ * For Example:
+ * FooBar -> "foo-bar"
+ * Foobar -> "foobar-tag" (adds suffix)
+ */
+const hyphaneteComponentName = (componentName) => {
+	const result = componentName.replace(/([a-z])([A-Z])/g, '$1-$2' ).toLowerCase();
+
+	return result.includes("-") ? result : `${result}-tag`;
+};
+
+/**
+ * Capitalizes first letter of string.
+ */
+const capitalizeFirstLetter = string => string.charAt(0).toUpperCase() + string.slice(1);
+
+/**
+ * Validates component name, enforcing PascalCase pattern - Button, MyButton.
+ */
+const PascalCasePattern = /^([A-Z]+)([A-Za-z]+)$/
```

Comment:
we can simplify this to: `/^[A-Z][A-Za-z]+$/` since we want to have at least 1 capital letter at the beginning of the string and groups are not used

Reply:
Yes, at first I thought we should force PascalCase like MyButton, but then I decided that it's too much